### ISSUE TITLE
[Bug] Clicking on already selected branch doesn't highlight it in the commit graph

### ISSUE THREAD

**Issue by timvermeulen** [ForkIssues/Tracker#2238]
Clicking on a branch in the sidebar highlights that branch's most recent commit in the commit graph. It doesn't do this when the branch is already selected, though. This can be slightly annoying when I've scrolled on the commit graph and want to go back to the branch that is still selected, and I often end up temporarily selecting a random other branch and then reselecting the branch I'm interested in just so I don't have to go look for it manually 😅

**Comment by markst** [comment]
Yes this is very frustrating. Particularly if you have multiple local branches tracking the same remote branches. 
What can happen is it will select a random local branch rather than the branch which is being selected.
Since there isn't any visual indication that the local branch is selected, you can end up committing to the wrong branch.

I have to switch to SourceTree.app in order to see what local branch I'm in fact selected!

**Comment by DanPristupov** [comment]
@markst it seems that your issue is absolutely unrelated to what OP reported.

**Comment by markst** [comment]
This issue tracker exists to help improve Fork.app through community collaboration.

So far, your responses to me have felt rude and unwelcoming. I shared a gif demonstrating a similar experience where selecting a Git branch does not check out the branch, which I believe is a confusing user experience.

I chose to comment here rather than open a new issue to reduce noise and administrative overhead. If you believe my report is unrelated, it would be more helpful to kindly point me to an existing issue rather than leaving an unconstructive comment.

**Comment by markst** [comment]
Since I received no response, I've created a new issue with relevant problem: https://github.com/fork-dev/Tracker/issues/2418

**Comment by DanPristupov** [comment]
Fixed in Fork 2.56
